src/app/frontend/deploy/createsecret.html

```diff
@@ -14,10 +14,11 @@
 limitations under the License.
 -->
 
-<md-dialog aria-label="Create a new secret" layout="column">
-  <md-dialog-content layout-padding>
+<md-dialog aria-label="Create a new secret" layout="column" layout-padding>
```

Comment:
Looks like a problem with merge. We need to use `<md-dialog-content>` and other elements that were changed in this commit. It needs to be reverted.